### ISSUE TITLE
Support for different camera resolutions

### ISSUE THREAD

**Issue by jghaanstra** [thanksmister/wallpanel-android#153]
**Is your feature request related to a problem? Please describe.**
I was only able to select a stream resolution of 640x480 while my (front) camera supports a higher resolution.

**Describe the solution you'd like**
Add more options in the camera select field with different solution (and of course actually show the stream in the selected resolution).

**Describe alternatives you've considered**
I'm currently using IP Webcam Pro which is a really nice solution for using your Android device as webcam. But it would be great if I would only need WallPanel for my needs and that WallPanel would support the basic feature of IP Webcam (like showing the video feed at the actualy resolution of the camera). But the current low resolution in WallPanel is now holding me back from ditching IP Webcam all together..

**Comment by mattstein111** [comment]
I believe the IP Webcam Pro (or non pro version, for that matter) supports being called for those (and other) functions from other Android apps. It may be easier to just rely on the installation of both for those features, although that still requires some integration. Referenced at the bottom of his 'cheats' page. http://ip-webcam.appspot.com/cheats.html

**Comment by jghaanstra** [comment]
> I believe the IP Webcam Pro (or non pro version, for that matter) supports being called for those (and other) functions from other Android apps. It may be easier to just rely on the installation of both for those features, although that still requires some integration. Referenced at the bottom of his 'cheats' page. http://ip-webcam.appspot.com/cheats.html

That seems to be a different solution. I was talking about https://play.google.com/store/apps/details?id=com.pas.webcam

**Comment by mattstein111** [comment]
Yes, that's the same solution. Follow the link to the Developers website that you flagged, and you'll see that it's the same one. Basically he's exposed most of it via API to other Android apps.

**Comment by thanksmister** [comment]
Yeah, we could expose the camera resolutions of the device and allow users to select them.  I wish the developer of IP Webcam pro would open source his project so I could also see how to produce the RSTP feeds.

**Comment by BoneheadFraggle** [comment]
> Yeah, we could expose the camera resolutions of the device and allow users to select them. 

For me MJPEG is totally fine, but I would really like to have the option to use higher resolutions when the device can handle it. 
Would that be possible?

**Comment by thanksmister** [comment]
We can try to expose camera resolution values and user can select, that's probably the best option at this point.
